src/main/resources/io/jenkins/plugins/designlibrary/Layouts/twoColumn.jelly

```diff
@@ -0,0 +1,24 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+  <l:layout title="Heading">
+    <l:side-panel>
+      <l:app-bar title="Heading" />
+
+      <l:tasks>
+        <l:task title="Root" href="./twoColumn" icon="symbol-jenkins" />
+        <l:task title="Second page" href="./twoColumn" icon="symbol-edit" />
+        <l:task title="Third page" href="./twoColumn" icon="symbol-details" badge="123" />
```

Comment:
This is wrong as the final version of https://github.com/jenkinsci/jenkins/pull/7084 does not have simple `int`-based badges.